app/controllers/people_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PeopleController < ApplicationController
+  def index
+    @people = Person.order(id: :desc)
+  end
+
+  def show
+    @person = Person.where(id: params[:id]).first
```

Comment:
Because you are on the show page, .first is not needed.  You could use something like Person.find_by(id: params[:id])